.github/workflows/maven.yml

```diff
@@ -8,11 +8,6 @@ on:
   schedule:
     - cron: '0 3 * * *'
   workflow_dispatch:
-    inputs:
-      reason:
-        description: "Trigger Comment (Optional)"
-        required: false
-        default: "Manual Workflow Run"
```

Comment:
Why remove this?

Reply:
Because the `inputs` is optional and we don't need it. Having the "workflow_dispatch:" is enough to get that "Run workflow" button so we can trigger builds manually.